sw/device/silicon_creator/rom_ext/rom_ext_start.S

```diff
@@ -147,18 +151,126 @@ _rom_ext_start_boot:
    * If an exception fires, the handler is conventionaly only allowed to clobber
    * memory at addresses below `sp`.
    */
-  la   sp, (_stack_end - 16)
```

Comment:
We don't need the stack yet so I think you could move this to the "Setup C runtime" part of the code and keep the `la`.

Reply:
Done!